pkg/services/ngalert/api/api_prometheus.go

```diff
@@ -36,12 +36,16 @@ func (srv PrometheusSrv) RouteGetAlertStatuses(c *models.ReqContext) response.Re
 	}
 	for _, alertState := range srv.manager.GetAll(c.OrgId) {
 		startsAt := alertState.StartsAt
+		valString := ""
+		if len(alertState.Results) > 0 && alertState.State == eval.Alerting {
+			valString = alertState.Results[0].EvaluationString
```

Comment:
There can be several evaluation results, right? This sets always the value of the first result (regardless if its state is `Alerting`) and ignores the value of the subsequent results. I wonder if it would be better to build just one `EvaluationString` (for all the results) and have it on the `alertState` instead?

Reply:
Presently only designed for classic conditions, which always a single result.

Excited to do Value with multi-dimensions (SSE) but it is a bit more involved as I have to extract values from multiple layers of the operation graph, within an expression (maybe), and across dataframes, so need to think of the best way to do it.